scripts/check_license.py

```diff
@@ -34,45 +33,45 @@
 
 # exclude files and dirs from license header check:
 EXCLUDE = [
+    ".coveragerc",
     ".devcontainer",
-    "eggs",
+    ".editorconfig",
     ".eggs",
-    "build",
-    "dist",
-    "develop-eggs",
-    "lib",
-    "lib62",
-    "parts",
-    "sdist",
-    "wheels",
-    "pip-wheel-metadata",
-    ".coveragerc",
     ".git",
     ".github",
     ".flake8",
-    ".gitignore",
-    ".pylintrc",
-    ".ruff.toml",
-    ".ruff_cache",
-    "example_config.yaml",
-    "config_schema.json",
-    "LICENSE",  # is checked but not for the license header
-    ".pre-commit-config.yaml",
-    "docs",
-    ".vscode",
+    ".gitignore" ".pre-commit-config.yaml",
```

Comment:
```suggestion
    ".gitignore", ".pre-commit-config.yaml",
```

Reply:
This should be fixed in the service template.